doc/source/reference/c-api/types-and-structures.rst

```diff
@@ -1072,13 +1073,13 @@ the internal structure of the iterator object, and merely interact
 with it through the use of the macros :c:func:`PyArray_ITER_NEXT` (it),
 :c:func:`PyArray_ITER_GOTO` (it, dest), or :c:func:`PyArray_ITER_GOTO1D`
 (it, index). All of these macros require the argument *it* to be a
-:c:type:`PyArrayIterObject *`.
+:c:type:`PyArrayIterObject *<PyArrayIterObject>`.
 
 
 PyArrayMultiIter_Type and PyArrayMultiIterObject
 ------------------------------------------------
 
-.. c:var:: PyArrayMultiIter_Type
+.. c:type:: PyArrayMultiIter_Type
```

Comment:
This is actually `c:data`, isn't it? PyArrayMultiIterObject` would be the C type.

Reply:
Indeed `PyArrayMultiIter_Type` is global variable.
But `.. c:data::` directive does not exist (while `:c:data:` roles exists. https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#the-c-domain). I'm not sure which directive is appropriate for global variables. (`.. data:`?)

Reply:
I think `c:var` is correct? Perhaps you need `PyTypeObject *PyArrayMultiIter_Type`